### PR TITLE
Create method to update EVC

### PR DESCRIPTION
**Pull request by macartur**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -172,6 +172,31 @@ def create_circuit(self):
...
+        """Update a circuit based on payload.
+
+        The EVC required attributes can't be updated.
+        """
+        data = request.get_json()
+        circuits = self.box.data if self.box else {}
+
+        if circuit_id not in circuits:
+            result = {'response': f'circuit_id {circuit_id} not found'}
+            return jsonify(result), 400
```
Comment (beraldoleal) [inline main.py]: Please, return 404.

File: models.py [hunk 2]
```diff
@@ -88,6 +90,20 @@ def __init__(self, **kwargs):
...
+    def update(self, *args, **kwargs):
+        """Update evc attributes.
+
+        This method will ignore all required attributes given.
+
+        Raises:
+            ValueError: message with error detail.
+
+        """
+        for attribute, value in kwargs.items():
```
Comment (beraldoleal) [inline models.py]: Please lets make safer here.. looking on `hasattr()` or `__dict__`.

Comment (beraldoleal) [reply models.py]: Where is the deploy code? Antonio will change the blueprint.

File: models.py [hunk 3]
```diff
@@ -88,6 +90,20 @@ def __init__(self, **kwargs):
...
+            if attribute in self.required_attributes:
+                raise ValueError(f'{attribute} can be updated.')
```
Comment (beraldoleal) [inline models.py]: can't be

File: models.py [hunk 4]
```diff
@@ -13,6 +13,8 @@
 class EVC(GenericEntity):
     """Class that represents a E-Line Virtual Connection."""
 
+    required_attributes = ['name', 'uni_a', 'uni_z']
```
Comment (beraldoleal) [inline models.py]: Please change a better name for this variable.

File: main.py [hunk 5]
```diff
@@ -172,6 +172,31 @@ def create_circuit(self):
...
+
+        try:
+            evc = self.evc_from_dict(circuits.get(circuit_id))
+            evc.update(**data)
+            self.save_evc(evc)
+            result = {evc.id: evc.as_dict()}
+            status = 200
+        except ValueError as exception:
+            result = "Bad request: {}".format(exception)
+            status = 403
```
Comment (hdiogenes) [inline main.py]: Should be 400

**Pull request closed without merging**